### PR TITLE
A bunch of components a bit better documented

### PR DESCRIPTION
**Pull request by oiorain**
# [title]

## Description
[add description]

## Demo
Example on : [deployed story link]

## API
[api details]

```jsx
<YourComponent />
```

## Voiceover

[voiceover gif]

### CODE REVIEW

File: packages/strapi-parts/src/Dialog/Dialog.stories.mdx [hunk 1]
```diff
@@ -18,7 +18,7 @@ import DeleteIcon from '@strapi/icons/DeleteIcon';
 
 # Dialog
 
-This is the doc of the `Dialog` component
+`Dialog` creates a box to confirm a delete action.
```
Comment (maevalienard) [inline packages/strapi-parts/src/Dialog/Dialog.stories.mdx:21]: It's not always a delete action. In most of the case yes, but I'd then remove the word "delete"

File: packages/strapi-parts/src/BaseRadio/BaseRadio.stories.mdx [hunk 2]
```diff
@@ -81,7 +79,7 @@ Description...
 
 ## LSize
 
-Description...
+A bigger size can be used.
```
Comment (oiorain) [inline packages/strapi-parts/src/BaseRadio/BaseRadio.stories.mdx:82]: i'm not sure the case works on Storybook. it seems the same size

File: packages/strapi-parts/src/Dialog/Dialog.stories.mdx [hunk 3]
```diff
@@ -36,30 +36,13 @@ import AlertWarningIcon from '@strapi/icons/AlertWarningIcon';
...
-          <Box padding={10} background="neutral0">
-            test overflow
-          </Box>
-          <Box padding={10} background="neutral0">
-            test overflow
-          </Box>
-          <Box padding={10} background="neutral0">
-            test overflow
-          </Box>
           <Dialog onClose={() => setIsVisible(false)} title="Confirmation" isOpen={isVisible}>
```
Comment (oiorain) [inline packages/strapi-parts/src/Dialog/Dialog.stories.mdx:46]: With the addition of the props, the overflow is taken care of so i removed the additional padding.

File: packages/strapi-parts/src/Popover/Popover.stories.mdx [hunk 4]
```diff
@@ -56,9 +58,7 @@ Description...
   </Story>
 </Canvas>
 
-## Popover
-
-Description...
+## Open by default
```
Comment (oiorain) [inline packages/strapi-parts/src/Popover/Popover.stories.mdx:61]: Not sure about this one..

File: packages/strapi-parts/src/Popover/Popover.stories.mdx [hunk 5]
```diff
@@ -91,7 +91,7 @@ Description...
 
 ## onReachEnd
 
-Description...
+You can define an action to be performed when the end of the popover is reached.
```
Comment (oiorain) [inline packages/strapi-parts/src/Popover/Popover.stories.mdx:94]: not sure about this one either.

**Review by maevalienard — CHANGES_REQUESTED** [review]
Awesome work, many thanks Marion! Your help is precious

File: packages/strapi-parts/src/BaseRadio/BaseRadio.stories.mdx (hunk 2 above)
Comment (maevalienard) [reply packages/strapi-parts/src/BaseRadio/BaseRadio.stories.mdx:82]: From what I remember, we decided to go for only one size. However, the wording is fine.

File: packages/strapi-parts/src/Popover/Popover.stories.mdx (hunk 4 above)
Comment (maevalienard) [reply packages/strapi-parts/src/Popover/Popover.stories.mdx:61]: What do you mean by "open by default"?

Comment (oiorain) [reply packages/strapi-parts/src/Popover/Popover.stories.mdx:61]: The popover seems to be open from the beginning
Screenshot 2021-09-29 at 14 18 35

Comment (maevalienard) [reply packages/strapi-parts/src/Popover/Popover.stories.mdx:61]: oh okay, that shouldn't probably be the case. Usually an open popover is triggered by an action first

File: packages/strapi-parts/src/Accordion/Accordion.stories.mdx [hunk 6]
```diff
@@ -168,7 +166,7 @@ The accordion component has two main states: collapsed and expanded. By default
 
 ## Accordion expanded
 
-Description...
+the Accordion component can also be open by default using the parameter `expanded={true}`.
```
Comment (HichamELBSI) [inline packages/strapi-parts/src/Accordion/Accordion.stories.mdx:169]: Can you please capitalize the first letter ? 🙏🏽

File: packages/strapi-parts/src/BaseCheckbox/BaseCheckbox.stories.mdx [hunk 7]
```diff
@@ -32,7 +30,7 @@ Description...
 
 ## Indeterminate
 
-Description...
+The checkbox can have an indeterminate state.
```
Comment (HichamELBSI) [inline packages/strapi-parts/src/BaseCheckbox/BaseCheckbox.stories.mdx:33]: `The BaseCheckbox ...`

File: packages/strapi-parts/src/Box/Box.stories.mdx [hunk 8]
```diff
@@ -20,7 +21,7 @@ import { Box } from '@strapi/parts/Box';
 
 ## Usage
 
-Description...
+Default Box component.
```
Comment (HichamELBSI) [inline packages/strapi-parts/src/Box/Box.stories.mdx:24]: The Box component is a very powerful component and it is used almost everywhere in the app. IMO we should give more details on that component. 

What do you thing about something like that:
```js
The Box component will allow you to create your containers according to the guidelines of the Strapi design system (border-radius, shadows, background-color, padding, border, etc).
```

**Review by HichamELBSI — CHANGES_REQUESTED** [review]

**Pull request merged**